roles/test-provision-apb/tasks/main.yml

```diff
@@ -19,3 +38,8 @@
   retries: 10
   delay: 20
   until: '"jvm_memory_bytes_used" in webpage.content'
+
+- name: Check that keycloak secret exists
+  shell: "oc get secrets -o name | grep {{ keycloak_secret_name }}"
+  register: kc_secret
+  failed_when: keycloak_secret_name not in kc_secret.stdout
```

Comment:
Missing curly brackets here? `{{ keycloak_secret_name }}`

Reply:
Actually they should not be used in `when` and `failed_when` statements.
Otherwise it's displaying warnings https://github.com/ansible/ansible/issues/22397

Reply:
I think that's fixed since 2.4, have you seen the warnings?

Reply:
It's also in the log from your yesterday's PR for metrics-apb: https://jenkins-wendy.ci.feedhenry.org/blue/organizations/jenkins/AeroGearCatalog%2Fmetrics-apb/detail/PR-26/2/pipeline/68#step-108-log-89

Haven't noticed it before, sorry. I've sent PR with fix: https://github.com/aerogearcatalog/metrics-apb/pull/27

Reply:
It's a good catch!